apps/job-service/src/jobs/dto/delayed-job.dto.ts

```diff
@@ -55,4 +55,17 @@ export class DelayedJobDto extends JsonApiAttributes<DelayedJobDto> {
     nullable: true
   })
   isAcknowledged: boolean | null;
+
+  @ApiProperty({
+    description: "The name of the delayedJob",
+    nullable: true
+  })
+  name: string | null;
+
+  @ApiProperty({
+    description: "The name of the related entity (e.g., site, project).",
+    nullable: true,
+    required: false
+  })
+  entityName?: string | null;
```

Comment:
I think the entity type should be included here too, even if it's not needed for the current feature.

Reply:
added